src/tests/ftest/object/create_many_dkeys.py

```diff
@@ -152,7 +125,7 @@ def write_a_bunch_of_values(self, how_many):
         print("destroy complete")
 
     @avocado.fail_on(DaosApiError)
-    @avocado.skip("Skipping until DAOS-1721 is fixed.")
+    @skipForTicket("DAOS-1721")
```

Comment:
If you decide to make the name change suggestion to this function, this will have to change.

Reply:
Yeah, I think for consistency with Avocado, I'm going to keep the camelCase.

Reply:
How is camel case consistent with avocado?  The avocado files, like https://github.com/avocado-framework/avocado/blob/master/avocado/core/test.py, use camel case only for classes.  Also isn't it simpler to correct the method name style than add in pylint enable/disable statements?

Reply:
https://avocado-framework.readthedocs.io/en/52.0/WritingTests.html#avocado-skip-decorators

Reply:
Ok, for the unittest/avocado decorators it makes sense